src/services/jwt.service.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken');
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '24h',
+  });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+}
+
+function signRefresh(user) {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY);
```

Comment:
Ensure that `process.env.JWT_REFRESH_KEY` is correctly set in your environment to securely sign and verify JWT refresh tokens.